app/src/tamilNadu/res/drawable/thanks_logo.xml

```diff
@@ -0,0 +1,84 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It's a large vector drawable, it would affect render performance. Is there a way we can simplify this? Can we just use a PNG or WEBP instead for this?

Reply:
https://upday.github.io/blog/vector_drawables_optimisation/